variables.tf

```diff
@@ -61,6 +61,13 @@ variable "member_cpu_count" {
   # Validation is done in the Terraform plan phase by the IBM provider, so no need to add extra validation here.
 }
 
+variable "member_host_flavor" {
+  type        = string
+  description = "Allocated host flavor per member."
```

Comment:
How is user supposed to know what values are supported here? Can you include some info on that?

Reply:
I will look for more documentation. What I have found so far has not helped.

Reply:
well I see the provider lists some options in https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/database#id